SETUP.md

```diff
@@ -104,102 +101,86 @@ External contributors can supply alternate placeholder values for secrets normal
 
 ### macOS
```

Comment:
From here on, I think its much easier to look at the markdown rendered versions before and after side by side, rather than trying to stare through the diff noise:
[Before](https://github.com/code-dot-org/code-dot-org/blob/1c8330e8668ba127aefb22368c9454941fa408cd/SETUP.md#macos) | [**After**](https://github.com/code-dot-org/code-dot-org/blob/seth/no-rosetta/SETUP.md#macos)